kustomize/plugin/qlik.com/v1/charthomefullpath/ChartHomeFullPath.go

```diff
@@ -2,11 +2,13 @@ package main
 
 import (
 	"fmt"
-	"io"
 	"io/ioutil"
+	"log"
 	"os"
 	"path"
 
+	"github.com/qlik-trial/kustomize-plugins/kustomize/utils"
```

Comment:
`github.com/qlik-trial/kustomize-plugins/kustomize/utils` should be `github.com/qlik-oss/kustomize-plugins/kustomize/utils`. may need to change in `go.mod` file as well

Reply:
addressed